server/src/internalClusterTest/java/org/opensearch/discovery/StableMasterDisruptionIT.java

```diff
@@ -73,8 +73,8 @@
 import static org.hamcrest.Matchers.equalTo;
 
 /**
- * Tests relating to the loss of the master, but which work with the default fault detection settings which are rather lenient and will
- * not detect a master failure too quickly.
+ * Tests relating to the loss of the cluster-manager, but which work with the default fault detection settings which are rather lenient and will
+ * not detect a cluster-manager failure too quickly.
  */
 @OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.TEST, numDataNodes = 0)
 public class StableMasterDisruptionIT extends OpenSearchIntegTestCase {
```

Comment:
Can we change this class name?

Reply:
The class is public, so I plan to rename it in next major version (3.0). It will be tracked in issue https://github.com/opensearch-project/OpenSearch/issues/1684

Reply:
Thank you for catching it! I forgot to rename it in my last scan. 😅
I changed the name in the new commit https://github.com/opensearch-project/OpenSearch/pull/2521/commits/9fbc35fb461810dc347a986ded45acd16bfda4fa